sdk/schemaregistry/azure-schemaregistry/tests/async_tests/recordings/test_schema_registry_async.test_schema_basic_async.yaml

```diff
@@ -7,60 +7,59 @@ interactions:
       Content-Length:
       - '201'
       Content-Type:
-      - application/json
-      Serialization-Type:
-      - Avro
+      - application/json; serialization=Avro
       User-Agent:
-      - azsdk-python-azureschemaregistry/1.0.0b3 Python/3.9.0 (Windows-10-10.0.19041-SP0)
+      - azsdk-python-azureschemaregistry/1.0.0b4 Python/3.9.0 (Windows-10-10.0.22000-SP0)
```

Comment:
this user-agent in the recording doesn't look correct.. should it be `1.0.0`?

I think just re-record with the latest version 1.0.0 should work :)

Reply:
ahhh, it was the _version.py, thanks!

Reply:
actually, I just checked and the _version is 1.0.0 and it's still not updating the recording?

Reply:
it was just my local env caching stuff, so it wasn't updating. worked when I used a fresh env!